capacitor.config.ts

```diff
@@ -21,6 +21,10 @@ const config: CapacitorConfig = {
       launchFadeOutDuration: 1000,
       backgroundColor: "#92FFC0"
```

Comment:
Is it working ?

Reply:
I don't think we can change it to css variable:
1.  As the [documentation](https://capacitorjs.com/docs/apis/splash-screen#configuration
) explained, format of this config is #RRGGBB or #RRGGBBAA

2. This setting has been applied to IOS and Android native code and I think it doesn't use css variable. 

IOS: https://github.com/ionic-team/capacitor-plugins/blob/main/splash-screen/ios/Sources/SplashScreenPlugin/SplashScreenPlugin.swift#L67

Android: https://github.com/ionic-team/capacitor-plugins/blob/main/splash-screen/android/src/main/java/com/capacitorjs/plugins/splashscreen/SplashScreenPlugin.java#L87

Reply:
Ok thanks for investigating that. We'll need to keep this in mind moving forward because this is going to be crucial when the time comes for us to build custom versions of the app with different colors. For the moment most of them are all in the color stylesheet except basic colors like `white`, but at some point we will actually need to dig into all that and make sure we have an easy way to control all colors from as few places as possible.